createdisk.sh

```diff
@@ -105,6 +106,25 @@ function generate_vbox_directory {
         >$destDir/crc-bundle-info.json
 }
 
+function generate_hyperkit_directory {
+    srcDir=$1
+    destDir=$2
+    tmpDir=$3
+
+    cp $srcDir/kubeadmin-password $destDir/
+    cp $srcDir/kubeconfig $destDir/
+    cp $srcDir/id_rsa_crc $destDir/
+    cp $srcDir/${CRC_VM_NAME}.qcow2 $destDir/
+    cp $tmpDir/vmlinuz-${kernel_release} $destDir/
+    cp $tmpDir/initramfs-${kernel_release}.img $destDir/
+
+    cat $srcDir/crc-bundle-info.json \
+        | ${JQ} ".nodes[0].diskImage = \"${CRC_VM_NAME}.qcow2\"" \
+        | ${JQ} ".storage.diskImages[0].name = \"${CRC_VM_NAME}.qcow2\"" \
+        | ${JQ} '.storage.diskImages[0].format = "qcow2"' \
+        >$destDir/crc-bundle-info.json
```

Comment:
I believe the diskImage metadata is already present in $srcDir/crc-bundle-info.json? (and you remove this in the next commit)